graphql/schema/types/performer.graphql

```diff
@@ -0,0 +1,196 @@
+enum GenderEnum {
+  MALE
+  FEMALE
+  TRANSGENDER_MALE
+  TRANSGENDER_FEMALE
+  INTERSEX
+}
+
+enum FakeTitsEnum {
+  YES
+  NO
+  NA
+}
+
+type Measurements {
+  cup_size: String
+  # TODO - should these be in cm to be consistent with height?
+  bust: Int
+  waist: Int
+  hip: Int
+}
+
+input MeasurementsInput {
+  cup_size: String
+  # TODO - should these be in cm to be consistent with height?
+  bust: Int
+  waist: Int
+  hip: Int
+}
+
+type Performer {
+  id: ID!
+  name: String!
+  disambiguation: String
+  aliases: [String!]!
+  gender: GenderEnum
+  urls: [URL!]!
+  birthdate: Date
+  ethnicity: String
+  country: String
+  eye_color: String
+  """Height in cm"""
+  height: Int
+  measurements: Measurements!
+  # TODO - is there a better desciptor for this - boob_job? - enhanced?
+  fake_tits: FakeTitsEnum
+  career_start_year: Int
+  career_end_year: Int
+  tattoos: [String!]
+  piercings: [String!]
```

Comment:
Ideally I also think piercings should be typed. 99% of piercings are in a handful of locations, and it would allow us to query based on navel piercing or whatever. Maybe add an optional description field.